rpm/poppler.spec

```diff
@@ -126,7 +127,7 @@ rm -f %{buildroot}%{_libdir}/*.la
 %defattr(-,root,root,-)
 %{_libdir}/libpoppler-qt5.so
 %{_libdir}/pkgconfig/poppler-qt5.pc
-%{_includedir}/poppler/qt5
+%{_includedir}/poppler/qt5/
 
 %files
 %defattr(-,root,root,-)
```

Comment:
If you feel like it, these %defattrs can be removed from the file entries. That's the default already, and we've been cleaning them up.

Reply:
Also make sure the -devel packages own `%{_includedir}/poppler`.